metadata-service/auth-filter/src/main/java/com/datahub/authentication/filter/AuthenticationFilter.java

```diff
@@ -142,7 +149,11 @@ private void buildAuthenticatorChain() {
           final Authenticator authenticator = clazz.newInstance();
           // Successfully created authenticator. Now init and register it.
           log.debug(String.format("Initializing Authenticator with name %s", type));
-          authenticator.init(configs);
+          if (authenticator instanceof DataHubTokenAuthenticator) {
```

Comment:
I'm confused - why even do this check? Why not just pass everyone a context map?

Reply:
I'm also thinking we should move DataHubTokenAuthenticator to being a "first-class" authenticator.

Reply:
Wired in by default.